### DISCUSSION TITLE
WatchFiles WARNING -> INFO

### DISCUSSION THREAD

**Discussion by gertcuykens** [Kludex/uvicorn#1701] [Q&A]
If you stop for a red light and your car stops the engine to safe fuel, how annoying would it be if your car would display a warning message to tell you it shutdown the car?

```WARNING:  WatchFiles detected changes in 'test.py'. Reloading...```

I believe a reload should be a INFO msg instead

```INFO:  WatchFiles detected changes in 'test.py'. Reloading...```

**Comment by euri10** [comment]
I dont disagree, good analogy, PR welcomed if anyone else ok :)

**Reply by gertcuykens** [reply]
https://github.com/encode/uvicorn/pull/1704

**Reply by Kludex** [reply]
Given that we are about to restart a process when this warning is raised, I think it's a valid level... But I don't really care about it. :man_shrugging:

Your call here @euri10 :pray:

**Reply by gertcuykens** [reply]
I understand but thats why I used the car analogy to make it more clear why expected behaviour shouldn't be a warning. For example you definitely don't want tire warning air pressure of a car in the same category as fuel saving.

**Reply by Kludex** [reply]
Since you mentioned it... I don't agree with the analogy.

**Reply by gertcuykens** [reply]
If you depend on a state that cant handle a expected reload the logic behind the specific state it self in your app should mention a warning after the info of a restart.

Example
INFO: WatchFiles detected changes in 'test.py'. Reloading...
WARNING: My App at line 50 got exception

**Reply by gertcuykens** [reply]
In addition in current version you also don't get warning if you restart server manually 
<img width="662" alt="image" src="https://user-images.githubusercontent.com/863731/195417254-41e21287-5511-4ee7-ab4e-6102a6eb8388.png">

<img width="347" alt="image" src="https://user-images.githubusercontent.com/863731/195416744-20e3be9d-c26e-4a9b-82e7-ba758d4fa75b.png">
So to be consistent it shouldn't do it either if you deliberately set it to auto reload

**Reply by Kludex** [reply]
Ah... Yeah... Then it makes sense.

EDIT: Or... Should we reconsider the level of the shutdown message? 👀

**Reply by gertcuykens** [reply]
Doh! XD No we do not want shutting down warning ... info is perfectly fine for everything you expect it to do. A user can always code a extra warning in the "on_shutdown" if they really want one
